mne/tests/test_cov.py

```diff
@@ -451,16 +453,20 @@ def test_compute_covariance_auto_reg():
     logliks = [c['loglik'] for c in covs]
     assert_true(np.diff(logliks).max() <= 0)  # descending order
 
+    methods = ['empirical',
+               'factor_analysis',
+               'ledoit_wolf',
+               # 'pca',  XXX FAILS
```

Comment:
@dengemann if you check out my branch you can just uncomment this line to see the failure